### ISSUE TITLE
Cheatsheet request: SVG

### ISSUE THREAD

**Issue by fabiobaser** [rstacruz/cheatsheets#569]

**Comment by chadwithuhc** [comment]
Do you mean SVG syntax? Or something else about SVG?

**Comment by bbuck** [comment]
Moving my comment from #646 

SVG, specifically path and it's d attribute can be quite cryptic. Info is easy to find in documentation but since you've got "cheatsheets" a quick reference would be great.

I'll take a look at what it takes to build a cheatsheet and may make an initial pass when I have some time.

**Comment by nicosemp** [comment]
I don't know if this is still in the works, but I was looking for an SVG cheat sheet right now.

I was watching a video that shows a bunch of possible SVG tags and their possible properties (see image below).
It would be nice to have them all laid out in a cheat sheet.

![image](https://user-images.githubusercontent.com/8838305/113520928-7d045100-9596-11eb-944e-b008eda8e9ad.png)

**Comment by fabiobaser** [comment]
@chadwithuhc Yes, I meant a cheatsheet containing the most useful tags and attributes. Like what @nicosemp proposed.
